src/mlpack/methods/ann/loss_functions/log_cosh_loss.hpp

```diff
@@ -0,0 +1,110 @@
+/**
+ * @file log_cosh_loss.hpp
+ * @author Kartik Dutt
+ *
+ * Definition of the Log-Hyperbolic-Cosine loss function.
+ *
+ * For more information, see the following paper.
+ * 
+ * @code
+ * article{
```

Comment:
If you have one, please add a permalink to this paper. 

Also, and much less importantly, it looks like most articles cited in this context have a bibtex shorthand intro (see [here for an example](https://github.com/mlpack/mlpack/blob/928aee24c75f4227b5c1702aeb0acdc9aaf486e0/src/mlpack/methods/ann/layer/elu.hpp#L84))

Reply:
I will add the url, however the authors are anonymous, Should I add 

> Anonymous2019

 ?